### PR TITLE
Samsung.DataMigration version 2.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106249)

### CODE REVIEW

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
```
Comment (Trenly) [inline manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.installer.yaml:5]:
```suggestion
PackageVersion: 2.0
```
This must match control panel. The version folder will need to be renamed

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
```
Comment (Trenly) [inline manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0
```
This must match control panel. The version folder will need to be renamed

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Samsung Data Migration Tool
```
Comment (Trenly) [inline manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml:8]:
```suggestion
PackageName: Samsung Data Migration
```
This must match control Panel

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
```
Comment (Trenly) [inline manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.yaml:5]:
```suggestion
PackageVersion: 2.0
```
This must match control panel. The version folder will need to be renamed

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.installer.yaml:5]: Please update file name and path to match this change.

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml (hunk 2 above)
Comment (stephengillie) [reply manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml:5]: Please update file name and path to match this change.

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml (hunk 3 above)
Comment (stephengillie) [reply manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml:8]: Please update file name and path to match this change.

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.yaml (hunk 4 above)
Comment (stephengillie) [reply manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.yaml:5]: Please update file name and path to match this change.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
> 2023-05-08 19:35:35.286 [CLI ] ShellExecute installer failed: 2147753984 
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230509023535

**Comment by stephengillie** [comment]
This package seems to install normally, but errors when launched: 
image

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
> 'manifests\s\Sa***ung\DataMigration\14.0.162', expected 
> 'manifests\s\Sa***ung\DataMigration\2.0'

**Comment by upintheairsheep** [comment]
Try a physical machine and also try in the VM with interactive installation

**Comment by stephengillie** [comment]
> Try a physical machine and also try in the VM with interactive installation

Interactive installation completed but the application gave the same error when launched. Unfortunately, all of our pipelines are VMs and none are bare metal.

**Comment by denelon** [comment]
Supporting custom hardware in validation is "Not Planned".
* https://github.com/microsoft/winget-pkgs/issues/199504

**Pull request closed without merging**